templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/index.vue

```diff
@@ -158,6 +217,36 @@ export default {
         }
       })
     },
+    getMenuName(index) {
+      return index === 0 ? this.node.title : this.menuGroups[index][0].node.title
+    },
+    getMenuIndexFromNodeId(nodeId) {

```

Comment:
I'm still pretty confused what the purpose of a menu index is and why we are not just using IDs for these? It requires all this code to convert between index and Ids

Reply:
The `menuIndex` in the `pageMenuData` object is for switching the lightbox to only show the contents of one menu at a time. In PageRows.vue, it uses the menuIndex to determine which menu group to display. If I just passed the nodeId, I would still have to perform the calculation to determine which menu the node belongs to.